go/storage/api/metrics.go

```diff
@@ -60,6 +61,19 @@ type metricsWrapper struct {
 	Backend
 }
 
+func (w *metricsWrapper) GetDiff(ctx context.Context, request *GetDiffRequest) (WriteLogIterator, error) {
+	start := time.Now()
+	it, err := w.Backend.GetDiff(ctx, request)
+	storageLatency.With(labelGetDiff).Observe(time.Since(start).Seconds())
```

Comment:
Errors should have lower latency and could corrupt your data. Therefore, it would be nice to also add a label stating whether the call returned and error or not (and maybe which type). But I guess this doesn't matter, as other functions use the same approach.

Reply:
Yeah noticed, decided to keep unified with the rest of the code. Can update in future if needed. I think this is fine, as long as one is aware of this.